js/common/lib/tensor-utils.ts

```diff
@@ -20,4 +20,13 @@ export interface TypedShapeUtils<T extends Tensor.Type> {
 }
 
 // TODO: add more tensor utilities
-export interface TypedTensorUtils<T extends Tensor.Type> extends Properties, TypedShapeUtils<T> {}
+export interface TypedTensorUtils<T extends Tensor.Type> extends Properties, TypedShapeUtils<T> {
+  /**
+   * creates an ImageData instance from tensor
+   *
+   * @param tensorFormat - Interface describing tensor instance - Defaults: RGB, 3 channels, 0-255, NHWC
+   * 0-255, NHWC
+   * @returns An ImageData instance which can be used to draw on canvas
+   */
+  toImage(tensorFormat?: ImageFromTensorOptions): ImageData;
```

Comment:
as discussed, `toImageData` may be better

Reply:
parameter should be `options` instead of `tensorFormat`

Reply:
Good point thanks for catching it - updated and I will push it soon